files/build/versions/default/versions-web

```diff
@@ -16,6 +16,8 @@ https://bootstrap.pypa.io/pip/2.7/get-pip.py==60e8267eb1b7bc71dc4843eb7bd294d3
 https://chrome-infra-packages.appspot.com/client?platform=linux-amd64&version=git_revision:e75c9bf286fbb31347379cb478df2a556ab185b1==9df7a96a4a4fb6e484027d80b3ee821c
 https://download.docker.com/linux/debian/gpg==1afae06b34a13c1b3d9cb61a26285a15
 https://github.com/aristanetworks/sonic-firmware/raw/6d0d1661d92a342acedb6839dba970ae5778b478/phy/phy-credo_1.0_amd64.deb==dd74acbb7bf979b01c1a89e2a628aaf3
+https://github.com/barefootnetworks/sonic-release-pkgs/raw/dev/bfnplatform_20210324_deb10.deb==b1f3087f05b82504130c5a7d428b9ef5
+https://github.com/barefootnetworks/sonic-release-pkgs/raw/dev/bfnsdk_20210324_deb10.deb==40a5468c7b7a003df6f4e6c2f9f98817
```

Comment:
@mssonicbld could you please review that this is compliment with your changes?
looks like both this signatures are required.

Reply:
@novikauanton , the versions are auto-managed, mssonicbld is the account of the automation tools. Please check-in the code together with version change. The version change will be removed by mssonicbld, because not reference detected.
We may need to add a feature to support the case to manage the version manually.

Reply:
@xumia thank for your comment, 
could you please share a link to the automation tool?
And how can the reference to bfnsdk be added to this tool?